### PR TITLE
Added missing template documents

### LINKED ISSUE #1: Create a full skeleton of ISTQB product package
To ease the development of a new ISTQB products there should be a full skeleton of documents ready with the documentation.

So following documents should be added in matter of folders with one MD file and a TeX file:

- `/release-notes`
- `/sample-exam`
- `/body-of-knowledge`
- `/accreditation-guidelines`

### PR DESCRIPTION
**Pull request by danopolan**
Closes #1

### CODE REVIEW

**Comment by danopolan** [comment]
> I will create a corresponding PR in the `istqb_product_template` repo, so that we can review the PDF outputs before merging.

That is what I wanted to consult, since I forgot about the fact of CI when defining Issue for splitting the repositories.
Would it be possible to verify changes in `istqb_product_base` by triggering downstream pipeline in `istqb_product_template`? Or do we have a design wrong and all documents should be moved to `istqb_product_template` repository?

**Comment by Witiko** [comment]
> Would it be possible to verify changes in `istqb_product_base` by triggering downstream pipeline in `istqb_product_template`? 

I don't think so, no.

> Or do we have a design wrong and all documents should be moved to `istqb_product_template` repository?

This will only add more busywork whenever we need to change both the code and the documentation.

> Or we do not run build in this base project as well?

We can run the CI in the base project. I will open a PR for that.

**Comment by danopolan** [comment]
> I will open a PR for that and then we can move on with this PR.

Great thx. I am trying to run the build locally as well, but run into issues.

**Comment by Witiko** [comment]
> > I will open a PR for that and then we can move on with this PR.
> 
> Great thx. I am trying to run the build locally as well, but run into issues.

We discussed this on WhatsApp.

Adding the symlink `istqb_product_base` -> `.` and adding the line `  - push` below the line `on:` in file `.github/workflows/compile.yml` should get the CI running in the repository `istqb_product_base`. This will allow us validate pull requests such as this one. We may want also want to tweak the DOCX rules, so that we don't unnecessarily convert YAML documents in the `schema/` subdirectory to DOCX.

If there is no rush, I will add the CI support by next Thurstday.

**Comment by Witiko** [comment]
In commit 30ff225, I enabled CI for the `istqb_product_base` repository. Running the CI on this PR shows that the document `body-of-knowledge/acknowledgments.md` that you input from `body-of-knowledge.tex` does not exist:

image

@danopolan, can you please add the missing document?

File: sample-exam-answers.tex [hunk 1]
```diff
@@ -0,0 +1,34 @@
+\documentclass{istqb}
```
Comment (Witiko) [inline sample-exam-answers.tex:1]: The CI assumes that every `*.tex` file has a directory `*/`. This allows us to skip producing an EPUB/HTML version for file `*.tex` if `*/NO_HTML` exists. Therefore, we should add symlinks `sample-exam-questions` and `sample-exam-answers` that point to the `sample-exam/` directory.

**Pull request merged**